digdag-tests/src/test/resources/acceptance/s3/s3_wait.dig

```diff
@@ -0,0 +1,6 @@
+
++wait:
+  s3_wait>: ${path}
```

Comment:
Not sure if fakes3 supports SSE-C. Maybe I should remove commands that are hard to test. Or maybe these tests should use the real S3 service.